docs/content/getting-started/install-sdk.mdx

```diff
@@ -166,6 +166,19 @@ go install github.com/openfga/cli@latest
 ### Manually
 Download the pre-compiled binaries from the [releases page](https://github.com/openfga/cli/releases).
 
+</TabItem>
+<TabItem value={SupportedLanguage.JAVA_SDK} label={languageLabelMap.get(SupportedLanguage.JAVA_SDK)}>
+
+You can find the Java package on [Maven Central](https://central.sonatype.com/artifact/dev.openfga/openfga-sdk).
+
+Using [Gradle](https://gradle.org/):
+
+<!-- markdown-link-check-disable -->
+
+```groovy
+implementation 'dev.openfga:openfga-sdk:0.3.0'
```

Comment:
In the repo we list a lot of variants for installing, I just picked gradle to list here but can add others as necessary

Reply:
I'd recommend adding Maven instructions. Maven and Gradle account for the vast majority of Java tooling https://www.jetbrains.com/lp/devecosystem-2023/java/#java_buildsystem Maven is used by ~71% of devs, Gradle by ~46% (declining year-over-year) and nothing else is over 10%.

Note: Gradle popularity is declining year over year

Reply:
Anyone using other tools (e.g. something like SBT for Scala or Leiningen for Clojure) is going to already know how to use their packaging/build tool to use a maven GAV. They might be pleasantly surprised if there were instructions here, but they generally won't expect something they can copy-paste.

Reply:
Have added Maven to the instructions, I'm not super thrilled about the static version but not sure how to deal with that short of doing something like fetching the latest SDK version on build :/

Reply:
static works for now, ideally we'd later add a process to the npm run build that would retrieve the latest versions